docs/releases/updating-npm-publishing-credentials.md

```diff
@@ -16,3 +16,12 @@ For more information on these visit the internal wiki page: <https://dev.azure.c
 1. `AZURESDKPARTNERDROPS_TENANT_ID`. This shouldn't change. This is the azure blob store tenant ID. This should be found on the [internal wiki page](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/1/Partner-Release-Pipeline).
 
 1. `AZURE_SDK_RELEASE_PIPELINE_DEVOPS_TOKEN`. **⚠ This will expire periodically and need updating ⚠**. This is a personal access token. An account with access to the [ADO release pipeline](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/1/Partner-Release-Pipeline) should create a new access token when the old one expires. To gain access to this release page you must belong to the appropriate security group; follow the instructions on the [internal wiki page](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/1/Partner-Release-Pipeline). For more information on how to create a personal access token see: [Use personal access tokens - Create a PAT](https://docs.microsoft.com/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops#create-a-pat).
+    * Request access to release pipelines through myaccess ([Request for project Azure SDK Partners](https://aka.ms/azsdk/join/azuresdkpartners))
+    * PAT Token should have the following scope:
+        * Release (Read, write, & execute)
+        * Code (Read, write)
+        * Build (Read, execute)
+     
+
+# Troubleshooting
+1. If you experience a 401 error, try rotating AZURE_SDK_RELEASE_PIPELINE_DEVOPS_TOKEN and rerun the job. If you still see the error, rotate the AZURESDKPARTNERDROPS_SERVICE_PRINCIPAL_KEY. You can get this key from the [kv](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/a18897a6-7e44-457d-9260-f2854c0aca42/resourceGroups/AzureSDKEng/providers/Microsoft.KeyVault/vaults/azuresdkpartnerdrops-kv). 
```

Comment:
Don't rotate the AZURESDKPARTNERDROPS_SERVICE_PRINCIPAL_KEY, just make sure to update it
```suggestion
1. If you experience a 401 error, try rotating AZURE_SDK_RELEASE_PIPELINE_DEVOPS_TOKEN and rerun the job. If you still see the error, update the AZURESDKPARTNERDROPS_SERVICE_PRINCIPAL_KEY. You can get this key from the [kv](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/a18897a6-7e44-457d-9260-f2854c0aca42/resourceGroups/AzureSDKEng/providers/Microsoft.KeyVault/vaults/azuresdkpartnerdrops-kv). 
```